.github/workflows/tests-build-image.yml

```diff
@@ -17,7 +17,8 @@ on:
 jobs:
   build-image:
     name: Build ${{ inputs.name }} image
-    runs-on: ["self-hosted", "linux", "arm64", "ubuntu-platform"]
+    # runs-on: ["self-hosted", "linux", "arm64", "ubuntu-platform"]
```

Comment:
How much slower it become? If much slower then let's revert it for now?

Reply:
moved docker to #2318